backend/worker/worker.go

```diff
@@ -526,7 +526,7 @@ func (w *Worker) excludeSession(ctx context.Context, s *model.Session, reason ba
 			"session_obj": s}).Warnf("error excluding session (session_id=%d, identifier=%s, is_in_obj_already=%v, processed=%v): %v", s.ID, s.Identifier, s.ObjectStorageEnabled, s.Processed, err)
 	}
 
-	if err := w.Resolver.OpenSearch.Update(opensearch.IndexSessions, s.ID, map[string]interface{}{
+	if err := w.Resolver.OpenSearch.UpdateSynchronous(opensearch.IndexSessions, s.ID, map[string]interface{}{
```

Comment:
Updated to use `UpdateSynchronous` per this slack [convo](https://highlightcorp.slack.com/archives/C02CJANPHQS/p1685660764377189) as well as the observation that the non async version, `Update`, can have unpredictable behavior (see #5454).